src/ast.rs

```diff
@@ -287,9 +286,8 @@ impl FuncCall {
     }
 
     pub fn check_builtin(&self) -> Option<BuiltIns> {
-        match self.class_name {
-            Some(_) => return None,
-            None => (),
+        if let Some(_) = self.class_name {
```

Comment:
This could even be written more tersely as `if self.class_name.is_some() {` I think.

Reply:
Fixed in a later commit.

Reply:
Oh, nevermind.  I'm reviewing by commit and see now that that change came in later.